app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/MainScaffold.kt

```diff
@@ -75,12 +92,28 @@ private fun MainScaffold(
                     icon = { NavigationIcon(isNavigationBar, navigationItem) },
                     label = { NavigationLabel(isNavigationBar, navigationItem) },
                     selected = currentDestination == navigationItem.destination,
-                    onClick = { navController.navigateToSelectedItem(navigationItem.destination) }
+                    onClick = { navigateToSelectedItem(navigationItem.destination) }
                 )
             }
         },
+        navigationSuiteColors = NavigationSuiteDefaults.colors(
+            navigationBarContainerColor = SwissTransferTheme.colors.navigationItemBackground,
+            navigationRailContainerColor = SwissTransferTheme.colors.navigationItemBackground,
+            navigationDrawerContainerColor = SwissTransferTheme.colors.navigationItemBackground,
+        ),
         layoutType = navType,
-        content = content,
+        content = {
+            if (navType == NavigationSuiteType.None) {
+                content()
+            } else {
+                Column {
+                    Box(modifier = Modifier.weight(1f)) {
+                        content()
+                    }
+                    HorizontalDivider(color = SwissTransferTheme.colors.divider)
```

Comment:
Can we use the surface color here ?

Reply:
The surface color wouldn't match the spec's colors. Why did you want to use surface specifically?